cns/logger/log.go

```diff
@@ -164,6 +164,26 @@
 	sendTraceInternal(msg)
 }
 
+// ResponseEx put request and response info together to help easier debug.
+func ResponseEx(tag string, request interface{}, response interface{}, returnCode int, returnStr string, err error) {
+	Log.logger.ResponseEx(tag, request, response, returnCode, returnStr, err)
+
+	if Log.th == nil || Log.DisableTraceLogging {
+		return
+	}
+
+	var msg string
+	if err == nil && returnCode == 0 {
+		msg = fmt.Sprintf("[%s] Sent %T %+v %T %+v.", tag, request, request, response, response)
+	} else if err != nil {
+		msg = fmt.Sprintf("[%s] Code:%s, %+v, %+v, %s.", tag, returnStr, request, response, err.Error())
+	} else {
+		msg = fmt.Sprintf("[%s] Code:%s, %+v, %+v.", tag, returnStr, request, response)
+	}
+
+	sendTraceInternal(msg)
```

Comment:
Why do we need this? Log.logger.ResponseEx will print to AI too, right?

Reply:
I didn't see either Log.logger.ResponseEx or Log.logger.Response prints to AI.
![image](https://user-images.githubusercontent.com/16043262/111388933-63de4380-866d-11eb-9340-2cba378fd9c7.png)

Reply:
After offline sync up, we'll keep it as it is for now. I'll make a separate PR to remove duplicate AI telemetry log populating issue.